### DISCUSSION TITLE
Tidak mendapatkan respon incoming_message webhook dari group

### DISCUSSION THREAD

**Discussion by shahib-mirbath** [kirimwa-id/forum#106] [Tanya Jawab]
Mas rio, tanya. Apakah webhook incoming_message hanya menerima dari selain group?
Terimakasih

**Comment by rioastamal** [comment]
@shahib-mirbath webhook incoming_message dapat menerima dari individual atau group. Check attribut `is_group_message` dan `group_id`.

**Reply by shahib-mirbath** [reply]
Saya mencoba menangkap semua respon, tp tidak ada.

Berikut script yg saya pake.
```$get = file_get_contents("php://input"); file_put_contents('respons/json_webhook_'.time().'.txt', $get);```

**Reply by shahib-mirbath** [reply]
@rioastamal 
Saya coba masukkan ke group lain (group asli, bukan group untuk tes), sudah dapat menerima respon incoming_message dari group.
Kok bisa gitu ya? Apa karena anggotanya hanya berdua?

**Reply by shahib-mirbath** [reply]
@rioastamal 
Lanjutan di atas.

Sudah dapat incoming_message dari group asli, tp group asli tersebut tidak ada di list group.
Apa ada jeda sinkron group ke kirimWA.id?

**Reply by shahib-mirbath** [reply]
@rioastamal 

Sudah tampil semua groupnya, tp setelah saya offlinekan dulu devicenya.

Apa memang gitu ya prosesnya sinkron groupnya?
🙏

**Reply by shahib-mirbath** [reply]
@rioastamal 

Belum dapat webhook incoming_message dari **ID Group** yg beda format.


ID Group paling atas

![20220305_182138.jpg](https://user-images.githubusercontent.com/46096681/156880928-0de7d335-03d5-4b56-bb7b-90c8c79ee399.jpg)

**Reply by rioastamal** [reply]
@shahib-mirbath, Apakah Group paling diatas dibuat dengan Multi Device beta?

**Reply by shahib-mirbath** [reply]
Dengan App terbaru dan Multi device beta.
Tapi saya coba buat group lagi dengan device yg tersambung ke kirimWA.id (tdk tergabung ke beta, tp pernah tergabung) hasil id groupnya sama dengan yg paling atas.

**Reply by msstoci** [reply]
> @rioastamal
> 
> Belum dapat webhook incoming_message dari **ID Group** yg beda format.
> 
> ID Group paling atas
> 
> ![20220305_182138.jpg](https://user-images.githubusercontent.com/46096681/156880928-0de7d335-03d5-4b56-bb7b-90c8c79ee399.jpg)

<img width="570" alt="Tangkapan Skrin 2022-04-12 pada 01 59 12" src="https://user-images.githubusercontent.com/11512375/162810486-aa7a730a-1791-4b0a-bdb2-b65881992def.png">

bener ini valid, ketika id_group nya berpola `nomor-nomor` itu bakalan dapat request dari webhook, sedangkan yg yg pola group id nya `nomorsaja` tidak mendapatkan balikan respon apa-apa
